### PR TITLE
chore: autopause for misconfigured entity

### PR DESCRIPTION
**Pull request by asdfryan**

### CODE REVIEW

**Review by lucasmarshall — CHANGES_REQUESTED** [review]
Why are we deleting so many docs here?

**Comment by asdfryan** [comment]
> Why are we deleting so many docs here?

Good question. I don't know what happened here other than maybe a bad merge? Let me try again.

**Review by lucasmarshall — APPROVED** [review]

File: packages/sync-workflows/workflows/run_object_sync.ts [hunk 1]
```diff
@@ -52,6 +52,9 @@ function getPauseReasonIfShouldPause(err: any): string | undefined {
   if (err.cause?.failure?.message.startsWith('No entity mapping found for entity')) {
     return err.cause.failure.message;
   }
+  if (err.cause?.failure?.message.startsWith('Additional field mappings are not allowed')) {
```
Comment (tomkit) [inline packages/sync-workflows/workflows/run_object_sync.ts:55]: Separate point: Errors we throw should be deterministic and the variable part of the message should be separate from the message
